src/blocks/cards.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  // height: 406px;
+
+  width: 200px;
+  background-color: $four-color;
```

Comment:
Ensure that the variable `$four-color` is defined in your SCSS files. If it's not defined, it will cause a compilation error.